.github/workflows/ci-lambda.yml

```
@@ -0,0 +1,98 @@
+name: CI - Python
+
+on:
+  pull_request:
+    types: [opened, synchronize, reopened]
+    paths:
+      - '**/*.py'
+      - '**/*requirement*.txt'
+
+  merge_group:
+    branches:
+      - master
```

Recommend just removing the branch filtering entirely. We (rarely) have PRs that target branches other than `master` (for long-lived feature branches, for example, although that practice is less favorable than feature flagging), and we would still want CI to run when merged to those branches. It doesn't hurt to make this CI run _more_ frequently than less frequently, either.

```suggestion
  merge_group:
```